src/test/fuzz/fuzz.cpp

```diff
@@ -77,6 +77,32 @@ void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target,
 
 static std::string_view g_fuzz_target;
 static const TypeTestOneInput* g_test_one_input{nullptr};
+bool ever_used_g_prng{false};
+
+inline void test_one_input(FuzzBufferType buffer)
```

Comment:
nit: Bit of a naming clash with `FuzzTarget::test_one_input`. :/

`FuzzTarget::test_one_input` is a subpar as a field name, but if you don't want to touch that:

```suggestion
inline void execute_one_input(FuzzBufferType buffer)
```